test/map/map.test.cpp

```diff
@@ -727,21 +727,6 @@ TEST(Map, AddLayer) {
     test::checkImage("test/fixtures/map/add_layer", test.frontend.render(test.map).image);
 }
 
-TEST(Map, WithoutVAOExtension) {
```

Comment:
can you please elaborate on why was this removed?

Reply:
From what I understand since we are on OpenGL ES 3.0 now so we have VAO Extensions, so we don't need this test anymore.

Reply:
https://www.khronos.org/opengl/wiki/Vertex_Specification#Vertex_Array_Object